registry/quilt_server/views.py

```diff
@@ -60,7 +60,8 @@
 PACKAGE_BUCKET_NAME = app.config['PACKAGE_BUCKET_NAME']
 PACKAGE_URL_EXPIRATION = app.config['PACKAGE_URL_EXPIRATION']
 
-DISALLOW_PUBLIC_USERS = app.config['DISALLOW_PUBLIC_USERS']
+ALLOW_PUBLIC_USERS = not app.config['DISALLOW_PUBLIC_USERS']
```

Comment:
Or `PREVENT_ANONYMOUS_ACCESS` if you want a different default. Neither are perfect but slowly getting clearer.

Reply:
I like DISALLOW_ANONYMOUS_USERS. We've been sticking to ALLOW/DISALLOW for config variables, and we should have access to both because I think we should have as a convention that config variables are false by default.